server/lib/activities.js

```diff
@@ -228,7 +228,7 @@ export default {
           case TransactionTypes.CREDIT:
             if (userTwitter) {
               tweet = encodeURIComponent(
-                `@${userTwitter} thanks for your ${formatCurrency(currency, recurringAmount)} donation to ${
+                `@${userTwitter} thanks for your ${formatCurrency(currency, recurringAmount)} contribution to ${
```

Comment:
Here I've skipped financial because we already have the amount before (`thanks for your $5 contribution`) so it's clear that it's a financial contribution.